packages/core/src/products/listing/redux/actions/doGetListing.js

```diff
@@ -30,7 +30,7 @@ import searchResultSchema from '../../../../entities/schemas/searchResult';
 export default getListing =>
   (slug, query, useCache = false, setListingHash = true, config) =>
   async (dispatch, getState, { getOptions = arg => ({ arg }) }) => {
-    const { productImgQueryParam, subfolder } = getOptions(getState);
+    const { subfolder } = getOptions(getState);
```

Comment:
Double check: is really wanted remove `productImgQueryParam` from this?

Reply:
It wasn't suppose to be removed, **fixed**!